### PR TITLE
Fix boolean parser error "IS NOT INSTANCE OF"

### LINKED ISSUE #1111: CHECK_59: IF object IS NOT INSTANCE OF class - abapOpenChecks boolean parser error
The following code sample leads to 'abapOpenChecks boolean parser error':

REPORT zcheck_59_test.

DATA: o TYPE REF TO zcl_excel.

IF o IS NOT INSTANCE OF zcl_excel.
  STOP.
ENDIF.

### PR DESCRIPTION
**Pull request by MiLiCLN**
closes #1111

### CODE REVIEW

File: src/parsing/zcl_aoc_boolean.clas.abap [hunk 1]
```diff
@@ -66,16 +66,20 @@ CLASS zcl_aoc_boolean IMPLEMENTATION.
...
 
     lv_token2 = io_tokens->get_token( 2 )-str.
     lv_token3 = io_tokens->get_token( 3 )-str.
     lv_token4 = io_tokens->get_token( 4 )-str.
+    lv_token5 = io_tokens->get_token( 5 )-str.
 
     rv_comparator = 0.
 
-    IF ( lv_token2 = 'IS' AND lv_token3 = 'NOT' )
+    IF ( lv_token2 = 'IS' AND lv_token3 = 'NOT' AND lv_token4 = 'INSTANCE' AND lv_token5 = 'OF' ).
```
Comment (MiLiCLN) [inline src/parsing/zcl_aoc_boolean.clas.abap:80]: Shorten to "IS NOT INSTANCE"?

Comment (larshp) [inline src/parsing/zcl_aoc_boolean.clas.abap:80]:
```suggestion
    IF lv_token2 = 'IS' AND lv_token3 = 'NOT' AND lv_token4 = 'INSTANCE' AND lv_token5 = 'OF'.
```

Comment (larshp) [reply src/parsing/zcl_aoc_boolean.clas.abap]: nah, dosnt matter

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**